timeless/access_control/views.py

```diff
@@ -14,7 +17,17 @@ def get(self, company_id):
      should take cls.resource value and method value from request.method
      and block the user if he hasn't privileges to access it.
     """
+
+    def check_permissions(self, *args, **kwargs):
+        """It raises exception if user has no permission"""
+        is_allowed = authorization.is_allowed(
+            method=request.method.lower(), resource=self.resource,
+            *args, **kwargs)
+
+        if not is_allowed:
+            abort(HTTPStatus.FORBIDDEN)
+
     def dispatch_request(self, *args, **kwargs):
-        method = getattr(self, request.method.lower(), None)
-        assert method is not None, "Unimplemented method %r" % request.method
+        if self.resource:
+            self.check_permissions(*args, **kwargs)
         return super(SecuredView, self).dispatch_request(*args, **kwargs)
```

Comment:
@vryazanov If you are already here pls fix if to `super().dispatch_request(*args, **kwargs)`

Reply:
@vladarefiev done